media-gfx/icoutils/icoutils-0.31.1.recipe

```diff
@@ -0,0 +1,73 @@
+# FIXME: extresso and genresscript won't work because of missing libwww-perl
+# and that won't compile because reasons
+SUMMARY="Tools for extracting and converting Windows icon and cursor files"
+DESCRIPTION="A set of command-line programs for extracting and converting \
+images in Microsoft Windows icon and cursor files (usually with the extension \
+.ico or .cur, but they can also be embedded in executables and DLL libraries."
+HOMEPAGE="http://nongnu.org/icoutils/"
+COPYRIGHT="1998-2017 Frank Richter"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="http://savannah.nongnu.org/download/icoutils/icoutils-$portVersion.tar.bz2"
+CHECKSUM_SHA256="26e29d3c78f25d4cdf402501ac0414c51a9a092daebf6c9dee3b837dee693093"
+PATCHES="icoutils-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	icoutil$secondaryArchSuffix = $portVersion
```

Comment:
If the recipes builds for x86_gcc2, x86 and x86_64, then you don't need any $secondaryArchSuffix.
This is because this package does not provide any library.

If, on the other side, t does not build on one (or more) of these primary architectures, then you should keep the $secondaryArchSuffix and add a SECONDARY_ARCHITECTURES with the appropriate archies, andfix ARCHITECTURES (with a ! where appropriate.)

Regarding the libpng16 vs libpng choice, this is a bit strange because both build fine on all architectures.
I think the problem is somewhere else.
I'll see if I can build it with libpng16 on x86_gcc2.

All the rest looks OK at first glance ;-)